examples/test-upgrade/cluster/old/main.tf

```diff
@@ -0,0 +1,30 @@
+resource "mongodbatlas_project" "test" {
+  name   = var.project_name
+  org_id = var.org_id
+}
+
+resource "mongodbatlas_cluster" "test" {
+  project_id                   = mongodbatlas_project.test.id
+  name                         = var.cluster_name
+  disk_size_gb                 = 100
+  num_shards                   = 1
+  replication_factor           = 3
+  provider_backup_enabled      = true
+  pit_enabled                  = true
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
```

Comment:
Just an FYI but we should get in the habit of changing these to the latest version of MDB, currently 4.4.   5.0 will come this summer.